### PR TITLE
Add updated_at to DBKnowledgeRepository database tables

### PR DESCRIPTION
**Pull request by danfrankj**
@matthewwardrop @NiharikaRay @earthmancash 

As discussed - adding an update field 

WARNING: if you already have a DB Repo you'll have to add a new column

### CODE REVIEW

File: knowledge_repo/repositories/dbrepository.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import logging
 import posixpath
 
+
```
Comment (matthewwardrop) [inline knowledge_repo/repositories/dbrepository.py]: Why did you add whitespace here?

Comment (danfrankj) [inline knowledge_repo/repositories/dbrepository.py]: accident - fixed
